portfolio/src/main/webapp/script.js

```diff
@@ -18,22 +18,28 @@ function loadPage() {
   getComments();
 }
 
-//TO DO Change the += format to use Node.appendChild()
 function getComments(){
-  fetch('/data').then(response => response.json()).then((messages) => {
+  var quantity = document.getElementById('commentCount');
+  const request = new Request('/data?quantity=' + quantity.value, {method:'GET'});
+  fetch(request).then(response => response.json()).then((messages) => {
     const commentContainer = document.getElementById("comment-container");
     var commentHTML = "";
     if (messages.length == 0){
-       commentHTML += "<p>There are no comments yet.</p>"
+       commentHTML += "<p>There are no comments yet. Why not post one yourself?</p>"
```

Comment:
Well done using `concat()` below! please consider using it here too for consistency

Reply:
Okay, thanks!